apps/web/src/pages/activities/components/ActivityStep.tsx

```diff
@@ -63,7 +63,7 @@ export const ActivityStep = ({ job, span = 4, isOld }) => {
                 <Timer width={16} height={16} />
               </When>
               <When truthy={status === JobStatusEnum.COMPLETED}>
-                <CheckCircle width="16" height="16" color={colors.success} />
+                <SuccessIcon width="16" height="16" color={colors.success} />
```

Comment:
As a global suggestion, it's preferred to name icons on what they are, and not where they will be used so we can can keep them reusable across the system.

Reply:
Done